docs/index.html

```diff
@@ -628,15 +635,15 @@ <h3 id="gettingstarted-requirements"><a class="link" href="#gettingstarted-requi
 please use the 1.0.x version of Spring Auto REST Docs.</p>
 </div>
 <div class="paragraph">
-<p>For Java 9 support, use <code>spring-auto-restdocs-json-doclet-jdk9</code> as doclet dependency.</p>
+<p>For Java 9/10/11 support, use <code>spring-auto-restdocs-json-doclet-jdk9</code> as doclet dependency.</p>
```

Comment:
Java 9+ would be more generic. Java 12 is already out.

Reply:
True, but they can always change Doclet things and it's safer to list only the ones we're building.